Terminal.Gui/Views/TextView.cs

```diff
@@ -303,10 +303,7 @@ internal static int CalculateLeftColumn (List<Rune> t, int start, int end, int w
 				if (rune == '\t') {
 					size += tabWidth + 1;
 				}
-				if (size > width) {
-					if (end == t.Count) {
-						col++;
```

Comment:
I don't feel confident reviewing the changes to column / offsets here as I'm not too familiar with how it works

Reply:
The cursor overlaps over the vertical or the horizontal scrollbar, when scrolling, due a bug in the size calculation.